src/axom/sidre/tests/sidre_mfem_datacollection.cpp

```diff
@@ -37,7 +38,7 @@ TEST(sidre_datacollection, dc_alloc_no_mesh)
 TEST(sidre_datacollection, dc_alloc_owning_mesh)
 {
   // 1D mesh divided into 10 segments
-  mfem::Mesh mesh(10);
+  auto mesh = mfem::Mesh::MakeCartesian1D(10);
```

Comment:
@white238 (and others) -- the `mfem::Mesh` constructors that generate Cartesian meshes are deprecated, and these new functions were added in mfem@4.3.  Please check that it is ok for your code if axom upgrades to the new API. (We discussed this offline).